src/util/login.js

```diff
@@ -0,0 +1,51 @@
+import * as firebase from 'firebase';
+
+/** Opens login prompt for user and redirects them to the home page if successful. */
```

Comment:
No these should be here, that's how JSDoc knows what comments to parse. http://usejsdoc.org/about-getting-started.html